workflow/envs/varlociraptor.yaml

```diff
@@ -2,4 +2,4 @@ channels:
   - conda-forge
   - bioconda
 dependencies:
-  - varlociraptor =1.8
\ No newline at end of file
+  - varlociraptor =2.0
```

Comment:
unfortunately we cannot merge, since varlociraptor 2.0 has bugs, so we have to wait for 2.1

Reply:
varlociraptor 2.0.0 is buggy, we have to set this to >=2.0.1 <2.1 or =2.0.1